assets/queries/terraform/azure/function_app_not_using_latest_tls_encryption_version/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	app := input.document[i].resource.azurerm_function_app[name]
+
+	app.site_config.min_tls_version != 1.2
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_function_app[%s].site_config.min_tls_version", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("'azurerm_function_app[%s].site_config.min_tls_version' is set to '1.2'", [name]),
+		"keyActualValue": sprintf("'azurerm_function_app[%s].site_config.min_tls_version' is not set to '1.2'", [name]),
+		"searchLine": common_lib.build_search_line(["resource", "azurerm_function_app", name, "site_config", "min_tls_version"], []),
+	}
+}
+
+CxPolicy[result] {
+	app := input.document[i].resource.azurerm_function_app[name]
+
+	not common_lib.valid_key(app, "site_config")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_function_app[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("'azurerm_function_app[%s].site_config' is defined and not null", [name]),
+		"keyActualValue": sprintf("'azurerm_function_app[%s].site_config' is undefined or null", [name]),
+		"searchLine": common_lib.build_search_line(["resource", "azurerm_function_app", name], []),
+	}
+}
```

Comment:
This rule is really necessary? Since the default value for TLS is 1.2, I think this check is not necessary...

Reply:
Great observation, Felipe! Thank you 🚀